packages/cli/src/api/client.ts

```diff
@@ -11,6 +13,13 @@ export const createClient: (options: ClientOptions) => AxiosInstance = ({
   apiToken: apiToken_,
 }) => {
   const apiToken = getApiToken(apiToken_);
+  if (!apiToken) {
+    const logger = log.getLogger(METICULOUS_LOGGER_NAME);
+    logger.error(
```

Comment:
I checked through all the commands, and I think they all require an API token (all hit BE endpoints that require auth)?